src/utils/device/api.ts

```diff
@@ -194,13 +195,14 @@ export class DeviceAPI {
 		});
 	}
 
-	public getLogStream(): Promise<Stream.Readable> {
+	public async getLogStream() {
 		const url = this.getUrlForAction('logs');
+		const sdk = getBalenaSdk();
 
-		// Don't use the promisified version here as we want to stream the output
-		return new Promise((resolve, reject) => {
-			const req = request.get(url);
+		const req = await sdk.request.stream({ url });
```

Comment:
Let's rename it from `req` to `stream` as it more correctly describe what this is